streaming/base/util.py

```diff
@@ -487,7 +489,10 @@ def retry(  # type: ignore
 
             num_tries = 0
 
-            @retry(RuntimeError, num_attempts=3, initial_backoff=0.1)
+            def clean_up():
+                print("cleaning up")
```

Comment:
Please use logging instead of `print` statement

Reply:
Oh this is not actual code, just the documentation code, but I can change it to be something more informative

Reply:
ok, nvm then!! You can keep the print statement.